NEWS.txt

```diff
@@ -94,10 +95,10 @@ Version 7.24 - 2022/07/22
   - rework menu labels
   - increase popup message timeouts
   - fix missing wind arrow InfoBox updates
-* file handling 
```

Comment:
Whitespace change?

Reply:
I guess my editor (Gedit) stripped some trailing spaces, I only added row 14...

Reply:
Then please fix your editor. If you want to add one line, then the commit diff should only show that one line, and not do lots of whitespace changes.

Reply:
Ok, done.
I really can't find any settings in Gedit about that behaviour, I think I'll file a bug.

Reply:
If you want to remove trailing spaces, that would be a reasonable thing to do (probably was added accidently by somebody many years ago), but that just shouldn't be hidden inside an unrelated commit.

Reply:
Yeah, sounds fine. Let me get this PR done and I'll make a new one to clean up trailing spaces :-)